<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DebugOptionsWidget</class>
    <widget class="QWidget" name="DebugOptionsWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>489</width>
                <height>201</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Debug</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <property name="fieldGrowthPolicy">
                        <enum>QFormLayout::ExpandingFieldsGrow</enum>
                    </property>
                    <property name="labelAlignment">
                        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                    </property>
                    <item row="0" column="0">
                        <widget class="QLabel" name="pluginLabel">
                            <property name="text">
                                <string>Debug Plugin:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QComboBox" name="pluginComboBox"/>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="argsLabel">
                            <property name="text">
                                <string>Program Arguments:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="debugArgs">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="esilstackAddr">
                            <property name="text">
                                <string>ESIL stack address:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLineEdit" name="stackAddr">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="0">
                        <widget class="QLabel" name="esilStackSize">
                            <property name="text">
                                <string>ESIL stack size:</string>
                            </property>
                        </widget>
                    </item>
                    <item row="3" column="1">
                        <widget class="QLineEdit" name="stackSize">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QWidget" name="widget_2" native="true">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <widget class="QCheckBox" name="esilBreakOnInvalid">
                        <property name="geometry">
                            <rect>
                                <x>0</x>
                                <y>0</y>
                                <width>469</width>
                                <height>24</height>
                            </rect>
                        </property>
                        <property name="text">
                            <string>Break esil execution when instruction is invalid (esil.breakoninvalid)</string>
                        </property>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
